.github/workflows/stale.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Close stale issues and PRs'
```

Comment:
This is not very helpful, if some issue is being opened longer than 180 days for example due to some missing functionality in Trino engine then we can easily forget about it and it will be closed. I understand that it can be commented or labelled to prevent removing but it's easy to overlook it in github notifications. We don't have so many issues and PRs in dbt-trino.

Reply:
ok let's remove that one